pom.xml

```diff
@@ -5,56 +5,47 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>1.5.22.RELEASE</version>
+        <version>2.6.2</version>
         <relativePath/> <!-- lookup parent from repository -->
     </parent>
     <groupId>org.isf</groupId>
     <artifactId>openhospital-api</artifactId>
-    <version>0.0.1</version>
+    <version>0.0.2</version>
     <name>openhospital-api</name>
-    <description>Openhospital Spring Boot REST API </description>
+    <description>Open Hospital Spring Boot REST API</description>
 
     <properties>
-        <oh.version>1.11.0</oh.version>
+        <oh.version>1.11.1</oh.version>
```

Comment:
```suggestion
        <oh.version>1.11.2</oh.version>
```

Reply:
@mwithi I don't think I can do that as the main Core repo still has 1.11.1 in it.

Reply:
true... but I'm merging soon 1.11.2